install/action.yaml

```diff
@@ -88,8 +95,8 @@ runs:
           if [ "$channel" = "nodefaults" ]
           then
             echo "Remove defaults channel"
-            conda config --remove channels defaults
-            conda config --env --remove channels defaults
+            conda config --remove channels defaults || echo nodefault
```

Comment:
Is *defaults* still there anyway?

Reply:
It is back again. I have kept the double pipe as I don't see it doing any harm.

Reply:
Jean-Luc told me once about *Chesterton’s Fence* and it struck me. You're right that it's not doing any harm now, but future us/contributors will be more hesitant to remove it if need be, will scratch their head looking at it, and worse will duplicate that all over the place as it seems required. Let's not do that :)